woocommerce/woocommerce-android
# Remove shared toolbar and add fragment specific toolbar

Closes: #10570 


### Description

This PR introduces individual toolbars for each fragment, replacing the shared toolbar previously used with `MainActivity`. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

Following fragments have been modified to contain its own toolbar
1. OrderListFragment
2. OrderDetailFragment
3. CustomerOrderNoteEditingFragment
4. AddOrderNoteFragment
5. PrintShippingLabelFragment
6. LabelFormatOptionsFragment
7. PrintShippingLabelInfoFragment
8. PrintShippingLabelCustomsFormFragment
9. ShippingLabelRefundFragment
10. CreateShippingLabelFragment
11. EditShippingLabelAddressFragment
12. EditShippingLabelPackagesFragment
13. EditShippingLabelPaymentFragment
14. ShippingCarrierRatesFragment
15. ShippingCustomsFragment
16. ShippingLabelAddressSuggestionFragment
17. ShippingLabelCreateCustomPackageFragment
18. ShippingLabelCreatePackageFragment
19. ShippingPackageSelectorFragment
20. SelectPaymentMethodFragment
21. PaymentsHubFragment
22. CardReaderOnboardingFragment

### Testing instructions

Please navigate through all the specified screens from the Order Detail screen in a two-pane layout, verifying the accuracy and functionality of each screen's toolbar and its action items. Additionally, confirm that the toolbars and their action items function identically on mobile phones, maintaining consistency across different device types.

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/cb2e0669-df2b-4e76-8b8b-fdac5747a54a
